benchmarks/scripts/cache_cleaning_coordinator_post_query.py

```diff
@@ -0,0 +1,80 @@
+import prestodb
```

Comment:
The cache cleaning should be done **before** (not after) each query cycle (cold+warm runs). And it should only be applied to TPC Power Tests.

Reply:
So in the current implementation, I added 2 hooks -
- post_query_scripts: run this script before this stage is started
- pre_stage_scripts  run this script after each query in this stage is complete
Which will eventually trigger the clean-up script, once when the overall query execution starts as part `pre_stage_scripts` and then using `post_query_scripts` after each query cycle (cold+warm runs) the same clean-up script would be executed. So by using both the hooks, I was running clean up script.

But now with this script name, I think it looks a little confusing. I will add one more hook named `pre_query_scripts` just call that before each query cycle (cold+warm runs) for each query and rename these clean up script from `post*` to `pre*`

Reply:
Hi @agrawalreetika I think the clean cache scripts should be called in every pre_query_cycle_scripts, not post_query_scripts and pre_stage_scripts. So we want this order suppose cold runs=1 and warm runs=1
clean cache, q1, q1, clean cache, q2, q2, clean cache, q3, q3, ...